docs/samples/database/external/README.md

```
@@ -0,0 +1,242 @@
+   ```
+
+4. **Check maas-api logs:**
+   ```bash
+   kubectl logs deployment/maas-api -n maas-api
+   ```
+
+### Unsupported Database
+
+Currently, only PostgreSQL is supported for external databases
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)